### PR TITLE
Create New certificate for ECDH exchange

### PR DESCRIPTION
**Pull request by cshari-zededa**
Creating this certificate to use in ECDH, for object level security

Signed-off-by: Hariharasubramanian C S <cshari@zededa.com>

### CODE REVIEW

File: pkg/pillar/scripts/device-steps.sh [hunk 1]
```diff
@@ -165,6 +165,11 @@ echo "$(date -Ins -u) device-steps: Starting upgradeconverter"
 status=$($BINDIR/upgradeconverter)
 echo "$(date -Ins -u) device-steps: upgradeconverter Completed. Status: $status"
 
+echo "$(date -Ins -u) device-steps: Creating additional security certificates"
+if ! $BINDIR/tpmmgr createCerts; then
```
Comment (eriknordmark) [inline pkg/pillar/scripts/device-steps.sh:169]: How did these get created up until now?

And what happens on an existing device where we have 4 of the 5 keys and only need to create the new one?
Are the functions robust and retrieve the existing 4 keys in that case?

Comment (cshari-zededa) [reply pkg/pillar/scripts/device-steps.sh]: These were created from RunAsService() method, I am moving it out to a separate call in device-steps.sh, to align with the objective of freezing /config partition after initial setup.  

If we have 4 out 5 keys, the method will create just the missing key (or cert).

**Comment by cshari-zededa** [comment]
> What remains to be done for this PR?
> 
> And see detailed question.

I had to do some more testing. I have completed it, and is ready for review now.

Thanks
Hari

File: pkg/pillar/scripts/device-steps.sh (hunk 1 above)
Comment (eriknordmark) [reply pkg/pillar/scripts/device-steps.sh]: ok

**Pull request merged**